package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backtrace-node",
-  "version": "1.2.1",
+  "version": "1.2.0",
```

Comment:
We should bump the version to 1.3.0 if we previously had 1.2.1

Reply:
previously was `1.1.1`, in one push i made it `1.2.1` instead of `1.2.0`, just correcting it here